plotting/map.py

```diff
@@ -395,11 +396,12 @@ def load_data(self):
                 quiver_bathymetry = overlays.bathymetry(
                     self.basemap, quiver_lat, quiver_lon)
 
-            self.data[np.where(self.bathymetry < depth_value)] = np.ma.masked
+            self.data[np.where(self.bathymetry < depth_value_map)] = np.ma.masked
             for d in self.quiver_data:
-                d[np.where(quiver_bathymetry < depth_value)] = np.ma.masked
+                temp1=np.where(quiver_bathymetry < depth_value)
```

Comment:
Is it possible to edit this to `d[np.where(quiver_bathymetry < depth_value)] = np.ma.masked`? I don't think this should change any behaviour, I just want to keep things concise.

Reply:
Other than that it's good to go.